mut/index/SnootyManifest.py

```diff
@@ -20,6 +20,7 @@ class ManifestEntry(TypedDict):
     code: dict
     preview: Optional[str]
     tags: List[str]
+    facets: Optional[Dict[str, any]]
```

Comment:
This needs to be `Any`, and must also be imported from typing on line 10

Reply:
great catch! updated